cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -38,6 +38,7 @@ def fields
         version: 'agent_version',
         status:  'status',
         initial: 'initial',
+        availability: 'availability',
```

Comment:
You mean something like this:
![image](https://user-images.githubusercontent.com/3205505/29074075-a74024e2-7c56-11e7-9df5-5278152af517.png)
@jakolehm WDYT?